music21/analysis/harmonicFunction.py

```diff
@@ -0,0 +1,290 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+# Name:         harmonicFunction.py
+# Purpose:      Mapping between Roman numeral figures and harmonic function labels
+#
+# Authors:      Mark Gotham
+#
+# Copyright:    Copyright © 2022 Michael Scott Cuthbert and the music21 Project
+# License:      BSD, see license.txt
+# ------------------------------------------------------------------------------
+
+from typing import Union
+import unittest
+
+from music21 import common
+from music21 import key
+from music21 import roman
+from music21 import scale
+
+from music21 import environment
+_MOD = 'analysis.harmonicFunction'
+environLocal = environment.Environment(_MOD)
+
+
+class FUNKTION(common.enums.StrEnum):
+
+    TONIKA_DUR = 'T'
+    TONIKA_DUR_PARALLELKLANG_MOLL = 'Tp'
+    TONIKA_DUR_GEGENKLANG_MOLL = 'Tg'
```

Comment:
Where there is a common English equivalent (TONIC_MAJOR)  I much prefer English.